### PR TITLE
[Term Entry] JavaScript Object Function: Object.isSealed()

### PR DESCRIPTION
**Pull request by phuongtrieu97coder**
### Description

-Added the Object.isSealed() term for JavaScript
-Closes Issue #4069 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/objects/terms/isSealed/isSealed.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  },
+  price: {
+    configurable: false,
+    writable: true,
+  },
+});
+console.log(Object.isSealed(fruits));
+```
+
+This will return the following output:
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isSealed/isSealed.md:71]:
```suggestion
The above code will return the following output:
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@phuongtrieu97coder Nice Work! I might have some more changes later, make these changes for now.

**Comment by phuongtrieu97coder** [comment]
@avdhoottt I've finished my review with all the changes. Please proceed to further review.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Looks good!! PR is ready for the second review.

**Review by dakshdeepHERE — APPROVED** [review]
Did some minor checks and this looks good for merge. 🚀

**Review by avdhoottt — APPROVED** [review]
Good to merge.

**Pull request merged**